src/components/ChatLog.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const chatComponents = props.entries.map((entry) => {
+        return (
+            <ChatEntry
+            key={entry.id}
+            id={entry.id}
+            sender={entry.sender}
+            body={entry.body}
+            timeStamp={entry.timeStamp}
+            liked={entry.liked}
+            updateChatData={props.updateChatData}
+            />
+        );
+    });
+
+    return (
+        <section>
+            {chatComponents}
+        </section>
+    )
```

Comment:
Instead of creating a variable `chatComponents` to reference the return value from calling map() on props.entries, it's more common to directly iterate over the chat entries in your component like this without using a variable:

```javascript
const ChatLog = ({entries, heartClick}) => {
    return (
     <section>
        {props.entries.map((entry) => (
            <ChatEntry 
                key={entry.id} 
                id={entry.id}
                sender={entry.sender}
                body={entry.body}
                timeStamp={entry.timeStamp}
                liked={entry.liked}
                updateChatData={props.updateChatData}
            />
        ))}
    </section>
    )
};